.storybook/stories/alert/alert.stories.ts

```diff
@@ -19,6 +19,7 @@ const defaultStory: Story = args => ({
       [clrAlertClosable]="clrAlertClosable"
       [clrAlertClosed]="clrAlertClosed"
       [clrAlertIcon]="clrAlertIcon"
+      [clrAlertLight]="clrAlertLight"
```

Comment:
I would rename this to `clrAlertLightweight` for consistency and to avoid confusion with the light theme.

Reply:
I tried to keep it simple but as I'm thinking...It can actually create some confusions. I'll change that. Thanks!